app/services/aapb/batch_ingest/pbcore_xml_mapper.rb

```diff
@@ -26,7 +26,7 @@ def prepare_annotations(annotations)
         final_annotations = []
 
         annotations.each do |anno|
-          annotation_type = find_annotation_type_id(anno.type)
+          annotation_type = anno.type ? find_annotation_type_id(anno.type) : nil
```

Comment:
if `find_annotation_type_id` is returning `nil` when `anno.type.empty?`, then I don't think you need this conditional.